app/api-v1/controllers/Recipe/__tests__/transactions.test.js

```diff
@@ -82,13 +96,14 @@ describe('recipe controller', () => {
       })
 
       it('does not perform any database calls and does not create transaction', () => {
-        expect(whereRecipeStub.calledOnce).to.equal(false)
-        expect(insertTransactionStub.calledOnce).to.equal(false)
+        expect(context.getAllRecipeTransactionsStub.calledOnce).to.equal(false)
       })
     })
+
     describe('if none transactions found', () => {
```

Comment:
This is a list API so this shouldn't be an error. It should instead just return an empty list

There is another test case you may wish to add to check if the recipe itself exists. If it doesn't that should be a `404 NotFoundError`

Reply:
makes sense